doc/mkdocs/docs/api/basic_json/erase.md

```diff
@@ -10,7 +10,8 @@ iterator erase(iterator first, iterator last);
 const_iterator erase(const_iterator first, const_iterator last);
 
 // (3)
-size_type erase(const typename object_t::key_type& key);
+template<typename KeyT>
+size_type erase(KeyT && key);
```

Comment:
I believe that erase wouldn't need to modify the key, so you'd want just const& here.

Reply:
I don't believe that you *should* need to do this, but if you're still getting ambiguities for this function, you could use std::enable_if to disable the function if the parameter type is the same as or converts to const_iterator.

Reply:
Will do the former.